build.sbt

```diff
@@ -231,6 +231,13 @@ lazy val zioHttpCli = (project in file("zio-http-cli"))
   .dependsOn(zioHttp)
   .dependsOn(zioHttpTestkit % Test)
 
+lazy val zioHttpHtmx = (project in file("zio-http-htmx"))
```

Comment:
Add it (`zioHttpHtmx`) to the `aggregatedProjects` list defined above as well, otherwise it won't be published

Reply:
Fixed, thank you.